### DISCUSSION TITLE
Welcome to p5 Better Discord Theme

### DISCUSSION THREAD

**Discussion by saysora** [saysora/p5#1] [General]

**Comment by ItsFonkyBoy** [comment]
Hi, thanks a lot for your work!, I would like to report that the user's profile page appears completely white when using the discord theme.

**Reply by saysora** [reply]
Thank you for letting me know. I'll add this to the list of issues that can be fixed.

**Comment by PhantomSparks** [comment]
hi can you add a quad column like this and yes am in a lot of servers
![Screenshot 2021-04-17 221824](https://user-images.githubusercontent.com/82745953/115135146-e2126a80-9fca-11eb-9e7a-41ffa2514ea0.png)

**Reply by saysora** [reply]
Thanks for the feedback!
At the moment, this isn't really something on my radar since I have for the most part been trying not to alter the UI in such ways. But I'll bear the suggestion in mind.

**Comment by Rweheretosuffer** [comment]
The new update to discord (today I think) broke one of my favourite parts of this theme, the user profile thingy, it used to look like this, but now it doesn't, I would really like it if you could repair this please, btw your theme is great, i absolutely love it
![userpopup](https://user-images.githubusercontent.com/84934190/119883506-891de680-bf27-11eb-97b8-eadc822582c8.png)

**Reply by saysora** [reply]
Thanks for letting me know.
I intend to jump in and re-implement some of the interfaces that will be changed, I'm just sorta waiting for Discord to make up their mind on their ui adjustments before jumping in to change things.

**Comment by layer02cyberia** [comment]
I made an account just to write this, but if you're still looking for ideas in this New Year, I had a good one for the chatting system..

![Persona-5-Chatting-on-Line](https://user-images.githubusercontent.com/97044595/147954102-db0fa4bc-0775-448e-bd36-97aa92e64ca3.png)
.

**Reply by saysora** [reply]
Ah yeah. The actual ui in the game is honestly so classic and nice. The only issue is implementing all of that into Discord with all their existing setup. Needless to say that undertaking would be massive. I'm not necessarily opposed, just being realistic.
